docs/logs-announcement.mdx

```diff
@@ -0,0 +1,16 @@
+---
+title: Improved logging and reporting is here!
+---
+
+# Big Update: Enhanced Logging and Test Reporting!
+
+We’ve made some major improvements to how you can monitor and analyze your tests! With Patrol 3.13.0 and later, you’ll get:
+
+ - Verbose logging: Test names are now displayed in real time as they’re executed!
+ - Detailed step reporting: See every action Patrol takes during your test execution, giving you deeper insights into the process.
+ - Flutter logs in console: Now you can access Flutter logs directly within the patrol test output, streamlining debugging and analysis.
+These enhancements will make it easier than ever to understand what's happening behind the code.
+
+For a full breakdown of these updates, check out the [Logs and test results][logs] page!
+
+[logs]: /logs
```

Comment:
And here it should be also `[Logs and test results]`

Reply:
but it works XD